### PR TITLE
Storage reqs

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Combines storage device content in Install Guide (from pubcloud reqs and general reqs) and moves them to hardware reqs.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/uyuni-project/uyuni-docs/issues/284

### CODE REVIEW

File: modules/installation/pages/hardware-requirements.adoc [hunk 1]
```diff
@@ -154,15 +160,95 @@ For more information about inter-server synchronization (ISS), see xref:administ
...
+You must set up the storage devices before you run the YaST {productname} setup procedure.
+
+{productname} requires three different volumes:
+
+* Database volume: [path]``/var/lig/pgsql``
+* Channel volume: [path]``/var/lib/spacewalk``
+* Cache: [path]``/var/cache``
+
+We recommend you use XFS as the filesystem type for all volumes.
+Additionally, consider using logical volume management (LVM) to manage the disks.
```
Comment (mbologna) [inline modules/installation/pages/hardware-requirements.adoc:188]: I am considering a SUSE Manager running in the public cloud here.
XFS is sure an option _if_ the volumes above are non-root volumes (e.g. AWS EBS).
LVM: I do not think that is feasible or recommended in a public cloud setup.

@jgleissner do you have any input in this regard?

Comment (jgleissner) [reply modules/installation/pages/hardware-requirements.adoc:188]: @mbologna XFS is recommended in the public cloud too. We even use it for root volumes. LVM is of limited use in the cloud and not recommended.

Comment (Loquacity) [reply modules/installation/pages/hardware-requirements.adoc:188]: OK, so the hardware reqs page should be advice common to all installations, and we can keep cloud-specific advice to the pubcloud reqs section. With that in mind, it seem that I can keep the hardware reqs page as-is, but update the pubcloud reqs page to read "do not use LVM"?

Comment (mbologna) [reply modules/installation/pages/hardware-requirements.adoc:188]: > OK, so the hardware reqs page should be advice common to all installations, and we can keep cloud-specific advice to the pubcloud reqs section. With that in mind, it seem that I can keep the hardware reqs page as-is, but update the pubcloud reqs page to read "do not use LVM"?

If the hardware requirements (this page) is common to all installations, then my suggestion is to be as general as possible: remove the LVM sentence from this and just add it to the "on-prem" documentation. So we do not have conflicting information between general and specific case setup.

Comment (Loquacity) [reply modules/installation/pages/hardware-requirements.adoc]: Updated.

**Pull request merged**